test/functional/tests/memory/test_memory_metadata_consumption.py

```diff
@@ -0,0 +1,196 @@
+    max_number_of_cores = 4096
+
+    # constants here correspond with OCF source code, where the same formula is used;
+    # they are partially empirical data so there are no proper/characteristic names for them
+    constant = Size(250, Unit.MebiByte)
+
+    cache_line_count = cache_size / cache_line_size.value
+    per_cache_line_metadata = Size(68) + 2 * cache_line_size.value / max_number_of_cores
+    amount_of_dram = constant + cache_line_count * per_cache_line_metadata
+
+    # maximum value is 110% of calculated size
+    metadata_max_size = 1.10 * amount_of_dram
+
+    return metadata_max_size
+
+
+@pytest.mark.parametrizex("cache_mode", CacheMode)
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_memory_metadata_consumption_empty_cache(cache_line_size, cache_mode):
+    """
+        title: Test for CAS RAM consumption for metadata.
+        description: |
+          Validate CAS RAM consumption for metadata depends on start command parameters.
+        pass_criteria:
+          - Successful CAS creation and cores addition.
+          - Memory consumption within limits.
+    """
+    with TestRun.step("Prepare drives and partitions for core devices."):
+        cache_disk = TestRun.disks["cache"]
+        cache_disk.create_partitions([cache_size])
+        cache_dev = cache_disk.partitions[0]
+
+        core_disk = TestRun.disks["core"]
+        parts = [cache_size / cores_count] * cores_count
+        core_disk.create_partitions(parts)
+        core_devices = core_disk.partitions
+
+    with TestRun.step("Start cache with configuration and add all core devices."):
+        free_mem_before_start = get_mem_available()
+
+        cache = casadm.start_cache(cache_dev, cache_mode, cache_line_size, force=True)
+
+        for dev in core_devices:
+            cache.add_core(dev)
+
+    with TestRun.step("Measure allocates size of metadata."):
+        used_memory = get_module_mem_footprint(cas_cache_module)
+        TestRun.LOGGER.info(f"Memory allocated: {used_memory}")
+
+    with TestRun.step(
+        "Compare allocated size with cache usage statistics " "and required consumption."
+    ):
+        free_mem_after_start = get_mem_available()
+
+        metadata_max_size = calculate_expected_metadata_footprint(cache.size, cache_line_size)
+        TestRun.LOGGER.info(f"Metadata max size: {metadata_max_size}")
+
+        validate_memory_consumption(cache, metadata_max_size, used_memory)
+
+        memory_delta = free_mem_before_start - free_mem_after_start
+        ram_change_status = is_memory_within_limit(
+            "Measured increment of RAM usage after start", metadata_max_size, memory_delta
+        )
+
+        if not ram_change_status:
+            TestRun.LOGGER.error(f"Memory measurements completed, found problems.")
```

Comment:
Error has been already reported by `is_memory_within_limit()`, so no need to print it one more time